source/quick-start-rails/download-and-install.txt

```diff
@@ -0,0 +1,120 @@
+
+====================
+Download and Install
+====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: ruby framework, odm, code example
+
+Prerequisites
+-------------
+
+To create the Quick Start application by using Rails 7, you need the
+following software installed in your development environment:
+
+- `{+language+}. <https://www.ruby-lang.org/en/documentation/installation/>`__
+  Rails requires {+language+}  v3.1.0 or later. Use the latest version
+  to prevent version conflicts.
+- `RubyGems package manager. <https://rubygems.org/pages/download>`__
+- A terminal app and shell. For MacOS users, use Terminal or a similar app.
+  For Windows users, use PowerShell.
+
+.. tip:: Rails 6 Tutorial
+
+   If you prefer to use Rails 6 to build your application, see the
+   :ref:`mongoid-getting-started-rails-6` guide.
+
+Download and Install the {+odm+} and Framework Gems
+---------------------------------------------------
+
+In {+language+}, packages are called **gems**.
+
+Complete the following steps to install and add the {+odm+} and Rails
+gems to your web application.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install Rails
+
+      Install the ``rails`` gem, which provides a command-line
+      interface to create an application skeleton and application
+      components.
+
+      Run the following command to install ``rails``:
+
+      .. code-block:: bash
+         
+         gem install rails
+
+   .. step:: Create a Rails skeleton app
+
+      Run the following commands to create a new Rails application
+      directory and enter it:
+
+      .. code-block:: bash
+
+         rails new {+quickstart-rails-app-name+} --skip-active-record
+         cd {+quickstart-rails-app-name+}
+
+      The ``--skip-active-record`` flag instructs Rails to not add
+      ``ActiveRecord`` as a dependency, as you will use {+odm+}
+      instead.
+
+      .. tip:: MacOS Installation Issue
+      
+         If you are using macOS, you might encounter issues when creating a
+         new Rails app during the automatic bundle installation step.
+         First, make sure that your macOS and `Xcode
+         <https://developer.apple.com/xcode/>`__ versions are up to
+         date. If you receive an error message similar to the following,
+         you must update or configure your build tools:
+      
+         .. code-block:: none
+            :copyable: false
+      
+            The compiler failed to generate an executable file.
+            ...
+            (RuntimeError) You have to install development tools first.
+      
+         Run the following commands to install Xcode command line tools:
+      
+         .. code-block:: bash
+      
+            xcode-select --install
+            xcodebuild -license accept
+
+         Then, try to run the ``bundle install`` command again.
+
+   .. step:: Add the {+odm+} gem
+
+      Open the ``Gemfile`` in your application and add the following
```

Comment:
Q: Is the file named just 'Gemfile'?

Reply:
its a proper noun for that file but yes it is also called Gemfile